### DISCUSSION TITLE
Updated / Simplified ReadME request

### DISCUSSION THREAD

**Discussion by hyzco** [ghiscoding/slickgrid-react#40] [General]
Hello, 

I am having trouble with Installation guide - it doesn't seem complete or missing because I am having trouble to configure it. It is very complex, would you be able to simplify it ? I request this to do the quick start.

Thank you

**Comment by ghiscoding** [comment]
The installation guide and other docs are in the Wikis, can you try to follow the [HOWTO - Step by Step](https://github.com/ghiscoding/slickgrid-react/wiki/HOWTO--Step-by-Step) instead. There's also the [React-Slickgrid-Demos](https://github.com/ghiscoding/slickgrid-react-demos) sample that might be helpful. I'd be happy to update any of the Wikis if you have better suggestions. Also as a side note, I'm not an actual React developer, someone else started the project and eventually left and transferred it to me, I made it work later.
